scripts/replace-all-mocks.py

```
@@ -0,0 +1,169 @@
+
+        # Check if import already exists
+        if "from config.mcp_config import MCPConfig" not in content:
+            # Add import after other imports
+            lines = content.split("\n")
+            import_added = False
+
+            for i, line in enumerate(lines):
+                if line.startswith("import ") or line.startswith("from "):
+                    # Find last import
+                    continue
+                elif import_added is False and i > 0:
```

Ruff (0.11.9)

Unnecessary `elif` after `continue` statement

Remove unnecessary `elif`

(RET507)